src/main/java/seedu/address/logic/parser/EditPersonCommandParser.java

```diff
@@ -14,22 +14,22 @@
 import java.util.Set;
 
 import seedu.address.commons.core.index.Index;
-import seedu.address.logic.commands.EditCommand;
-import seedu.address.logic.commands.EditCommand.EditPersonDescriptor;
+import seedu.address.logic.commands.EditPersonCommand;
+import seedu.address.logic.commands.EditPersonCommand.EditPersonDescriptor;
 import seedu.address.logic.parser.exceptions.ParseException;
 import seedu.address.model.tag.Tag;
 
 /**
  * Parses input arguments and creates a new EditCommand object
```

Comment:
```suggestion
 * Parses input arguments and creates a new EditPersonCommand object
```
Would be good if you could rename the commands in comments as well.